### DISCUSSION TITLE
Merging and unmerging

### DISCUSSION THREAD

**Discussion by scargill** [IceWhaleTech/CasaOS#1048] [General]
You have added a feature to MERGE drives in CASAOS - but NOWHERE can I find any discussion on how to UNMERGE them. I've merged 2 small SSDs into the main CASA emmc. I've now bought a new large SSD and need to unmerge the drives, preferably without losing all the information and setup - so that IU can then fit the Zimaboard out with the new large SSD. Any ideas as to where such information would be?

**Comment by crippledrainbowz** [comment]
Has this been figured out?
